src/components/OpenWith/OpenWith.js

```diff
@@ -332,4 +338,5 @@ class OpenWith extends PureComponent<Props, State> {
     }
 }
 
+export type OpenWithProps = Props;
```

Comment:
Can this be exported in line with its declaration above and be renamed on import?

Reply:
I think that would work, I'm following the pattern of the other elements that are used in content explorer like preview and the sidebar.